opentracing-api/pom.xml

```diff
@@ -25,8 +25,8 @@
         <version>0.1.0-SNAPSHOT</version>
     </parent>
 
-    <artifactId>opentracing</artifactId>
-    <name>OpenTracing</name>
+    <artifactId>opentracing-api</artifactId>
```

Comment:
@michaelsembwever got it... I have been thinking forward (maybe the word is "fantasizing") about `brave` eventually just supporting OT out of the box, so rather than `opentracing-brave`, users will just list `brave` as their dependency... at that point, plain old `opentracing` makes more sense.

Anyway, let's go with what you have. Just wanted to understand the rationale a bit better.